From a2f7171a59a5f2a277c871b2cf2cd1d157ed14f5 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Wed, 28 Nov 2018 18:02:12 +0900
Subject: [PATCH] makefile: add DSPP_VF_MODULE to compile as default

This patch is to add DSPP_VF_MODULE to activate compilation of spp_vf as
default.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/vf/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/vf/Makefile b/src/vf/Makefile
index 42c9037..ac100bb 100644
--- a/src/vf/Makefile
+++ b/src/vf/Makefile
@@ -22,6 +22,7 @@ CFLAGS += -DALLOW_EXPERIMENTAL_API
 CFLAGS += $(WERROR_FLAGS) -O3 -MMD
 CFLAGS += -I$(SRCDIR)/../shared
 CFLAGS += -I$(SRCDIR)/common
+CFLAGS += -DSPP_VF_MODULE
 #CFLAGS += -DSPP_DEMONIZE
 #CFLAGS += -DSPP_RINGLATENCYSTATS_ENABLE
 
-- 
2.7.4

